docker: fix default network name in docker-compose.yml

emersion opened the pull request:
If omitted, the project name is automatically deduced from the parent directory name. This was working fine for container names because we were explicitly defining their names, but the network name was left unspecified. osrdyne default config depends on the network name being "osrd_default", but that wasn't the case when the parent directory wasn't named "osrd".

Explicitly set the project name to fix this. Drop explicit names for container names, since the default values should be fine now.

Khoyo commented:
I'm fine with specifying the project but can we keep the container names? Otherwise they get named eg. `osrd-postgres-1` which isn't pretty

emersion committed:
docker: fix default network name in docker-compose.yml

If omitted, the project name is automatically deduced from the
parent directory name. This was working fine for container names
because we were explicitly defining their names, but the network
name was left unspecified. osrdyne default config depends on the
network name being "osrd_default", but that wasn't the case when
the parent directory wasn't named "osrd".

Explicitly set the project name to fix this.

Signed-off-by: Simon Ser <contact@emersion.fr>

emersion commented:
> I'm fine with specifying the project but can we keep the container names? Otherwise they get named eg. osrd-postgres-1 which isn't pretty

Oh, indeed! Dropped that change.
